libraries/WiFi/examples/WiFiScan/WiFiScan.ino

```diff
@@ -19,30 +19,69 @@ void setup()
 
 void loop()
 {
-    Serial.println("scan start");
+    Serial.println("Scan start");
 
-    // WiFi.scanNetworks will return the number of networks found
+    // WiFi.scanNetworks will return the number of networks found.
     int n = WiFi.scanNetworks();
-    Serial.println("scan done");
+    Serial.println("Scan done");
     if (n == 0) {
         Serial.println("no networks found");
     } else {
         Serial.print(n);
         Serial.println(" networks found");
+        Serial.println("Nr | SSID            | RSSI | CH | Encryption");
```

Comment:
Suggest changing this to accommodate 32 character SSID maximum to:
`Serial.println( " # | SSID                             | RSSI | CH | Encryption" );`
Suggest adding the following after the above to "pretty up" the output table:
`Serial.println( "---------------------------------------------------------------" );`

Reply:
I will also suggest to add `BSSID` column to differentiate between APs with the same `SSID`

Reply:
@me-no-dev Should I increase the column size? The limitation was on purpose because it is easier to read a table row without a huge space between the columns. Most SSIDs were shorter than 15 chars on my test setup.

Reply:
Since this is an example, I recommend printing the whole SSID. I agree a shorter SSID field looks better, but that can be customized for each application.  This is the header I use for printing WiFi list (debug use only):
```
Serial.println(" #                             SSID  BSSID              RSSI  Ch  Authmode  Rate" );
  Serial.println("-----------------------------------------------------------------------------------" );
```
I print the full SSID with `sprintf(ptr, "%32s  ", AP->ssid[0] != 0 ? (const char *)AP->ssid : "<hidden>");`.